content/en/methods/oauth.md

```diff
@@ -108,7 +108,7 @@ redirect_uri
 : {{<required>}} String. Set a URI to redirect the user to. If this parameter is set to urn:ietf:wg:oauth:2.0:oob then the token will be shown instead. Must match one of the `redirect_uris` declared during app registration.
 
 scope
-: String. List of requested OAuth scopes, separated by spaces (or by pluses, if using query parameters). If `code` was provided, then this must be equal to the `scope` requested from the user. Otherwise, it must be a subset of `scopes` declared during app registration. If not provided, defaults to `read`.
+: String. List of requested OAuth scopes, separated by spaces (or by pluses, if using query parameters). If `code` was provided, this will be ignored and the response will always include the `scope` requested from the user. Otherwise, it must be a subset of `scopes` declared during app registration. If no `code` _and_ no `scope` is provided, it defaults to `read`.
```

Comment:
Thanks for looking into that! Maybe it should be made more explicit that it is for `client_credentials` only? e.g.:

```suggestion
: String. When `grant_type` is set to `client_credentials`, the list of requested OAuth scopes, separated by spaces (or pluses, if using query parameters). Must be a subset of the scopes requested at the time the application was created. If omitted, it defaults to `read`. Has no effect when `grant_type` is `authorization_code`.
```